.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: python
 
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=pypy
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
     - TOXENV=coverage
 install:
     - pip install tox
-    - if [ "$TOXENV" = 'py34' ]; then pip install coveralls; fi
 script:
     - tox -e $TOXENV
-after_success:
-    - if [ "$TOXENV" = 'py34' ]; then coveralls; fi
```

Comment:
While there is no more Python 3.4, when you remove the coveralls call, coverage won't be shown any more -that would be a pitty.

Could you please re-add the logic, but for e.g. Python 3.8? Thank you!

Reply:
I changed for the "coverage" environment which makes it more flexible. It will not depend on the Python version anymore as this one will be taken from the matrix.